aten/src/ATen/native/cuda/Loops.cuh

```diff
@@ -37,16 +37,16 @@ struct needs_dynamic_casting<func_t, 0> {
   }
 };
 
-template<int N>
+template<int N, int num_outputs = 1>
```

Comment:
number of output can be obtained from `iter`

Reply:
thank you.